src/VisualizationObject.ts

```diff
@@ -12,6 +12,7 @@ export namespace VisualizationObject {
 
     export type BucketItem = IMeasure | IVisualizationAttribute;
 
+    export type VisualizationObjectExtendedFilter = VisualizationObjectFilter | IMeasureValueFilter;
```

Comment:
Wouldn't it be better if the measure value filter is already within `VisualizationObjectFilter` interface and therefore there is no some "extended" filter `VisualizationObjectExtendedFilter`?

Reply:
This is what we meant to do at the beginning, but we noticed that some types and methods require the type IMeasureValueFilter to be excluded, for example IMeasureDefinition.